Python-packages/covidcast-py/covidcast/plotting.py

```diff
@@ -0,0 +1,82 @@
+"""This contains the plotting and geo data management methods for the COVIDcast signals."""
+
+import os
+
+import geopandas as gpd
+import pandas as pd
+
+from covidcast import covidcast
+
+SHAPEFILE_PATHS = {"county": "../shapefiles/county/cb_2019_us_county_5m.shp",
+                   "state": "../shapefiles/state/cb_2019_us_state_5m.shp"}
+
+CURRENT_PATH = os.path.dirname(os.path.realpath(__file__))
```

Comment:
It's probably better to use [pkg_resources](https://setuptools.readthedocs.io/en/latest/pkg_resources.html#resourcemanager-api) to get the path to data distributed through the package, since I dunno what happens for eggs and all the zillion ways Python packages are distributed, and whether exact paths are preserved for them all

Reply:
Added, though I haven't worked with packaging static files in this way before so worth double checking. For some reason the package installs and runs fine regardless of if I include `package_data` in the[ setup.py](https://setuptools.readthedocs.io/en/latest/setuptools.html#including-data-files). Need to dig a bit deeper to fully understand whats going on.

The other thing I learned is that geopandas can read from URLs as well as zip files, so you can actually run the following. I know we discussed not pulling data at runtime and I think that's still the right idea, but just wanted to point this out as an FYI.
`gpd.read_file("https://www2.census.gov/geo/tiger/GENZ2019/shp/cb_2019_us_county_20m.zip")`

Reply:
As long as we can run the tests successfully, I suspect it's fine; my main concern is with the distribution of eggs and wheels and such.